src/zope/schema/_compat.py

```diff
@@ -2,23 +2,14 @@
 
 PY3 = sys.version_info[0] >= 3
 
-try:
-    from collections import OrderedDict
-except ImportError:  # pragma: no cover
-    from ordereddict import OrderedDict
+from collections import OrderedDict
```

Comment:
We could have modules that use OrderedDict import it directly instead of through `_compat`.